client/lib/upgrades/actions/domain-management.js

```diff
@@ -296,6 +296,12 @@ function applyDnsTemplate( domainName, template, variables, onComplete ) {
 	} );
 }
 
+function getDnsTemplateConflicts( domainName, template, variables, onComplete ) {
```

Comment:
I wasn't sure if we were trying to put all of the calls to `wpcom` in one place...which is why I put it here. In this case, I don't think I need an action creator since I'm not updating state anywhere else in the app (the entire Domain Connect async flow lives on this one page). I'm not sure that's correct though. Any idea who I might solicit additional input from on this point?